hazelcast/src/main/java/com/hazelcast/scheduledexecutor/ScheduledTaskStatistics.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.scheduledexecutor;
+
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+import com.hazelcast.spi.annotation.Beta;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Statistics and timing info for a {@link IScheduledFuture} accessible through {@link IScheduledFuture#getStats()}
+ */
+@Beta
+public interface ScheduledTaskStatistics extends IdentifiedDataSerializable {
+
+    /**
+     * Returns how many times the task was ran/called.
+     *
+     * @return The numbers of runs
+     */
+    long getTotalRuns();
+
+    /**
+     * @return The monotonic creation time the task.
+     */
+    long getCreatedAt();
+
+    /**
+     * @return The monotonic time of when the task was scheduled in for the very first time.
+     */
+    long getFirstRunStart();
+
+    /**
+     * @return The monotonic time of when the task was scheduled in last.
+     */
+    long getLastRunStart();
+
+    /**
+     * @return The monotonic time of when the task was last scheduled out.
+     */
+    long getLastRunEnd();
+
+    /**
+     * Returns the duration of the task's last execution.
+     *
+     * @param unit The time unit to return the duration in.
+     * @return The total duration of the task's last execution.
+     */
+    long getLastRunDuration(TimeUnit unit);
```

Comment:
Why does this method have a unit as parameter and the others not. I'm fine with both approaches; but currently it isn't consistent.